packages/discovery-react-components/src/components/SearchFacets/components/__tests__/FieldFacets.test.tsx

```diff
@@ -615,4 +618,212 @@ describe('FilterFacetsComponent', () => {
+          const euFacetValue = fieldFacetsComponent.getByText('eu (57158)');
+          const bostonFacetValue = fieldFacetsComponent.queryByText('boston (57158)');
+          const pennsylvaniaFacetValue = fieldFacetsComponent.queryByText('pennsylvania (57158)');
+          const quantityFacetValue = fieldFacetsComponent.queryByText('$299 (32444)');
+          const ibmFacetValue = fieldFacetsComponent.queryByText('ibm (138993)');
+          expect(pittsburghFacetValue).toBeDefined();
+          expect(usFacetValue).toBeDefined();
+          expect(euFacetValue).toBeDefined();
+          expect(bostonFacetValue).toBe(null);
+          expect(pennsylvaniaFacetValue).toBe(null);
+          expect(quantityFacetValue).toBe(null);
+          expect(ibmFacetValue).toBe(null);
+        });
+
+        test('can be toggled to show more than 5 facet values of expanded category', () => {
+          const { fieldFacetsComponent } = setupResult;
+          const locationCategoryHeader = fieldFacetsComponent.getByText('Location');
+          fireEvent.click(locationCategoryHeader);
+          const showMore = fieldFacetsComponent.getByTestId('show-more-less-Location');
+          fireEvent.click(showMore);
+          const bostonFacetValue = fieldFacetsComponent.getByText('boston (57158)');
+          const pennsylvaniaFacetValue = fieldFacetsComponent.getByText('pennsylvania (57158)');
+          expect(bostonFacetValue).toBeDefined();
+          expect(pennsylvaniaFacetValue).toBeDefined();
+        });
+
+        test('can expand multiple categories and collapse them again', () => {
+          const { fieldFacetsComponent } = setupResult;
+          const locationCategoryHeader = fieldFacetsComponent.getByText('Location');
+          const organizationCategoryHeader = fieldFacetsComponent.getByText('Organization');
+          fireEvent.click(locationCategoryHeader);
+          fireEvent.click(organizationCategoryHeader);
+          let pittsburghFacetValue = fieldFacetsComponent.queryByText('pittsburgh (57158)');
+          let ibmFacetValue = fieldFacetsComponent.queryByText('ibm (138993)');
+          expect(pittsburghFacetValue).toBeDefined();
+          expect(ibmFacetValue).toBeDefined();
+          fireEvent.click(locationCategoryHeader);
+          pittsburghFacetValue = fieldFacetsComponent.queryByText('pittsburgh (57158)');
+          ibmFacetValue = fieldFacetsComponent.queryByText('ibm (138993)');
+          expect(pittsburghFacetValue).toBe(null);
+          expect(ibmFacetValue).toBeDefined();
+          fireEvent.click(organizationCategoryHeader);
+          ibmFacetValue = fieldFacetsComponent.queryByText('ibm (138993)');
+          expect(ibmFacetValue).toBe(null);
+        });
+      });
+
+      describe('when clearing facets', () => {
+        test('can select facet values within one category and clear them', () => {
+          const { fieldFacetsComponent, performSearchMock } = setupResult;
```

Comment:
Could you please add some comments in some of the longer tests?

Reply:
Yes, done, thanks!